ext/chart_kit/ang/crmChartKit/chartKitTypes.service.js

```diff
@@ -1,47 +1,60 @@
-(function(angular, $, _) {
+(function (angular, $, _) {
   "use strict";
 
   // Provides pluggable chart types for use in the chart_kit display and admin components
-  angular.module('crmChartKit').factory('chartKitTypes', (chartKitPie, chartKitRow, chartKitStack, chartKitComposite) => {
+  angular.module('crmChartKit').factory('chartKitTypes', (
+    chartKitPie,
+    chartKitRow,
+    chartKitStack,
+    chartKitComposite,
+    chartKitSeries
+  ) => {
+
     const ts = CRM.ts('chart_kit');
 
     return [
-        {
-            key: 'pie',
-            label: ts('Pie'),
-            icon: 'fa-pie-chart',
-            service: chartKitPie,
-        },
-        {
-            key: 'row',
-            label: ts('Row'),
-            icon: 'fa-bar-chart fa-rotate-90',
-            service: chartKitRow
-        },
-        {
-            key: 'line',
-            label: ts('Line'),
-            icon: 'fa-line-chart',
-            service: chartKitStack
-        },
-        {
-            key: 'bar',
-            label: ts('Bar'),
-            icon: 'fa-bar-chart',
-            service: chartKitStack
-        },
-        {
-            key: 'area',
-            label: ts('Area'),
-            icon: 'fa-area-chart',
-            service: chartKitStack
-        },
-        {
-            key: 'composite',
-            label: ts('Combined'),
-            icon: 'fa-bar-chart',
-            service: chartKitComposite
-        },
+      {
+        key: 'pie',
+        label: ts('Pie'),
+        icon: 'fa-pie-chart',
+        service: chartKitPie,
+      },
+      {
+        key: 'row',
+        label: ts('Row'),
+        icon: 'fa-chart-bar',
+        service: chartKitRow
+      },
+      {
+        key: 'line',
+        label: ts('Line'),
+        icon: 'fa-line-chart',
+        service: chartKitStack
+      },
+      {
+        key: 'bar',
+        label: ts('Bar'),
+        icon: 'fa-chart-columnt',
+        service: chartKitStack
+      },
+      {
+        key: 'area',
+        label: ts('Area'),
+        icon: 'fa-chart-area',
+        service: chartKitStack
+      },
+      {
+        key: 'composite',
+        label: ts('Combined'),
+        icon: 'fa-bar-chart',
```

Comment:
Is this icon right?

Reply:
No it's not. Sorry it looks like I hadn't pushed up the final bits I did on this last night.

That should be updated now - along with UI improvement to choose the Grouping column separately from the X-axis column.